build.xml

```diff
@@ -59,14 +381,72 @@
 		</exec>
 	</target>
 
-	<!-- to option copy file to webapps :) -->
+	<target name="clear-tomcat" description="Delete Tomcat workdir">
+		<delete dir="${env.CATALINA_HOME}/work/Catalina/localhost"/>
+		<delete file="${env.CATALINA_HOME}/webapps/ScadaBR.war"/>
+		<delete dir="${env.CATALINA_HOME}/webapps/ScadaBR"/>
+	</target>
+
+	<target name="build" description="Perform project build workflow" depends="
+		init-core,
+		init-static-dependencies,
+		init-ui-dependencies,
+		build-core,
+		build-ui
```

Comment:
Junit tests should also be run in build.

Reply:
In current version Junit test are not included into build phrase.

Reply:
Yes that's right, but it would be nice to add junits to the process, I mean, the programmer would have to make some move to not fire them, by default they should be fired.

Reply:
In the latest [commit](https://github.com/SCADA-LTS/Scada-LTS/pull/1626/commits/fbfe86facfe7279283a90d802e0d3b18903b67b6) I've added the Unit-Tests as default to the build process but I have also added additional tasks that can start the application omitting the tests. See the updated table at the PR description.